kubetest/e2e.go

```diff
@@ -483,6 +488,21 @@ func DumpClusterLogs(location string) error {
 	return finishRunning(exec.Command("./cluster/log-dump.sh", location))
 }
 
+func ChartsTest() error {
+	// Clone the charts repository
+	if err := finishRunning(exec.Command("git", "clone", "--depth", "50",
```

Comment:
It would be great if `bootstrap.py` could handle this instead. @fejta can bootstrap clone multiple repos yet?